docs/guide/built-in-rules-callback.md

```diff
@@ -1,20 +1,20 @@
-# `Callback` - a wrapper over `callable`
+# `Callback` - a wrapper around `callable`
 
-This rule allows validation of current attribute value (but not limited to it) with an arbitrary condition set within a 
-callable. The benefit of it is that creation of a separate custom rule and handler is not required.
+This rule allows you to validate the current attribute value (but not limited to it) with an arbitrary condition within a  
+callable. The advantage of this type of validator is that there is no need to create a separate custom rule and handler.
 
 A condition can be within:
 
 - Standalone callable function.
 - Callable class.
-- DTO method.
+- DTO (data transfer object) method.
 
-The downside drown from using standalone functions and DTO methods is a lack of reusability. So they are mainly useful 
+The downside of using standalone functions and DTO methods is a lack of reusability. So they are mainly useful 
 for some specific non-repetitive conditions. Reusability can be achieved with callable classes, but depending on other
-factors (the need for additional parameters for example), it might be a good idea to create a full-fledged custom rule 
-with a separate handler instead.
+factors (the need for additional parameters for example), it might be a good idea to create a full-fledged
+[custom rule](creating-custom-rules.md) with a separate handler instead.
```

Comment:
Suggest to put the link to the end for consistency.

Reply:
Would you please elaborate?

Reply:
I mean leave just "[custom rule]" and place "[custom rule]: creating-custom-rules.md" at the end. See examples for existing links.